source/reactjs/src/Components/Shared/SharedLayout.ts

```diff
@@ -89,6 +89,15 @@ export const AccessibilityIcon = styled.div`
     background-image: url('/icons/accessibility-white.png');
     background-size: 16px 16px;
     font-size: 16;
+
+
+    @media screen and (forced-colors: active) and (prefers-color-scheme: light) {
+        background-image: url('/icons/ic_fluent_accessibility_24_regular.svg');
+    }
+
+    
+    
+
```

Comment:
Can we please remove the extra whitespace?

Reply:
removed whitespace